galaxy/wrapper/computeMatrix.xml

```diff
@@ -85,12 +79,14 @@
                 help="Label to use in the output."/>
         </repeat>
 
-        <param name="scoreFile" format="bigwig" type="data"
```

Comment:
We are using it already here: https://github.com/fidelram/deepTools/blob/release-1.6/galaxy/wrapper/deepTools_macros.xml#L292